.github/workflows/docker-publish.yml

```diff
@@ -0,0 +1,50 @@
+name: Docker publish to docker.io
+
+on:
+  push:
+    paths:
+      - .github/workflows/docker-publish.yml
+      - 'packages/**'
+      - 'docker-bin/**'
+      - 'package.json'
+      - 'pnpm-*.yaml'
```

Comment:
master branch uses yarn, in this case would be `yarn.lock`. Remember the master branch structure is completely difference, this looks like you took it from the branch `5.x`. At this point we want only cover `master === 4.x`.

Reply:
```
  push:
    paths:
      - .github/workflows/docker-publish.yml
      - 'src/**'
      - 'conf/**'
      - 'types/**',
      - 'docker-bin/**'
      - 'bin/**' 
      - 'package.json'
      - 'yarn.lock'
```